### ISSUE TITLE
How do you train SEResNet?

### ISSUE THREAD

**Issue by zhongqiu1245** [open-mmlab/mmclassification#302]
Hi, dear authors!
Sorry to bother you.
When you were training SE-ResNet50, did you start training from scratch or did you use pre-training weights loaded with resnet50(load_from = resnet50 or pretrain = resnet50)?
Thank you!

**Comment by mzr1996** [comment]
Hello, we have uploaded our [original train config](https://github.com/open-mmlab/mmclassification/blob/master/configs/seresnet/seresnet50_b32x8_imagenet.py`), you can simply check or use it.

**Comment by zhongqiu1245** [comment]
Hi @mzr1996
The link you uploaded is invalid(404). Can you upload it again?
Thank you!

**Comment by zhongqiu1245** [comment]
I have seen your config.py about SEResNet before, and it looks like you are training from scratch, because `load_from=None` in the config file.
Am I right?

**Comment by mzr1996** [comment]
> 
> 
> I have seen your config.py about SEResNet before, and it looks like you are training from scratch, because `load_from=None` in the config file.
> Am I right?

I have updated the link.
Right, it's trained from scratch if the `load_from` and `pretrained` config is `None`

**Comment by zhongqiu1245** [comment]
By the way, what does the value of step in lr_config mean?  How to calculate these values?

**Comment by zhongqiu1245** [comment]
Any news？

**Comment by mzr1996** [comment]
> Any news？

Hello, the `step` key means at which epoch, we will decay the learning rate, and the default decay ratio `gamma=0.1`.
For example, in config
```python
lr_config = dict(policy='step', step=[50, 100])
```
After the 50th epoch, the learning rate will become `0.1 × Lr` and after the 100th epoch, the learning rate will become 
`0.01 × Lr`

**Comment by zhongqiu1245** [comment]
谢谢你的回复
我知道step的作用了
但是我还有个疑惑，就是如何去step的数值？以及设置多少个step？凭检验吗？还是有什么经验公式可循？
